subsys/bluetooth/host/conn_internal.h

```diff
@@ -610,3 +610,9 @@ void bt_conn_tx_processor(void);
  * - unref the conn when popping the conn from the slist
  */
 void bt_conn_data_ready(struct bt_conn *conn);
+
+/* Returns the number of bt_conn_tx objects allocated for ACL data packets. */
+size_t bt_conn_tx_acl_count(void);
+
+/* Returns the number of bt_conn_tx objects allocated for ISO data packets. */
+size_t bt_conn_tx_iso_count(void);
```

Comment:
Since these will always return `CONFIG_BT_BUF_ACL_TX_COUNT` and `CONFIG_BT_ISO_TX_BUF_COUNT` respectively, it could be argued that defining them as functions are superfluous. I assume that the code size overhead is worth the added semantics

Reply:
> Since these will always return CONFIG_BT_BUF_ACL_TX_COUNT and CONFIG_BT_ISO_TX_BUF_COUNT respectively

I wouldn't mix Kconfigs with `conn_tx` and `iso_tx` arrays. These functions return size of both arrays. The fact that they match Kconfigs is because Host don't need more than what it can send over HCI to Controller.

Reply:
Used `CONFIG_BT_BUF_ACL_TX_COUNT` and `CONFIG_BT_ISO_TX_BUF_COUNT` directly. The change now shouldn't add overhead if warnings are disabled.